lib/Path/Tiny.pm

```diff
@@ -1454,19 +1454,23 @@ sub mkpath {
 
     path("foo.txt")->move("bar.txt");
 
-Move the current path to the given destination path using Perl's
-built-in L<rename|perlfunc/rename> function. Returns the result
-of the C<rename> function (except it throws an exception if it fails).
+Moves the current path to the given destination using L<File::Copy>'s
+C<move> function. Upon success, returns the C<Path::Tiny> object for the
+newly moved file.
```

Comment:
I think it would helpful to users to repeat `File::Copy::move`'s semantics here re directories and partial results on error.

Reply:
good idea